pallets/parachain-staking/src/mock.rs

```diff
@@ -46,7 +46,7 @@ construct_runtime!(
 	{
 		System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
 		Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>},
-		Stake: stake::{Pallet, Call, Storage, Config<T>, Event<T>},
+		ParachainStaking: parachain_staking::{Pallet, Call, Storage, Config<T>, Event<T>},
```

Comment:
This is a large part of the diff. It was required because the migration is applied to the storage item with pallet prefix `ParachainStaking` and the migration wasn't working on the same storage item in the pallet unit test context because the pallet unit tests used the pallet prefix `Stake`. The only way to test the migration in the unit test context is to change the pallet prefix for the unit tests.